site/conf.py

```diff
@@ -5,6 +5,11 @@
 DOC_ROOT = Path(__file__).parent.absolute()
 sys.path.extend([DOC_ROOT.as_posix(), DOC_ROOT.parent.as_posix()])
 
+# Add _ext directory to Python path for custom Sphinx extensions
+_ext_path = str(DOC_ROOT.parent / '_ext')
+if _ext_path not in sys.path:
```

Comment:
Maybe it belongs to `shared_conf.py` and the main site is not the only one to use it, so lets just move it to there.